openquake/risklib/scientific.py

```diff
@@ -1446,21 +1422,25 @@ class LossesByPeriodBuilder(object):
 
     :param insured_losses: insured losses flag from the job.ini
     """
-    def __init__(self, return_periods, loss_dt, num_rlzs, eff_time):
+    def __init__(self, return_periods, loss_dt, weights, num_events, eff_time):
```

Comment:
Please update the docstring (that even before these changes was not aligned with the signature)

Reply:
Done